src/main/java/eutros/multiblocktweaker/crafttweaker/predicate/ICandidates.java

```diff
@@ -0,0 +1,18 @@
+package eutros.multiblocktweaker.crafttweaker.predicate;
+
+import crafttweaker.annotations.ZenRegister;
+import eutros.multiblocktweaker.crafttweaker.gtwrap.interfaces.IBlockInfo;
+import eutros.multiblocktweaker.crafttweaker.gtwrap.interfaces.IBlockWorldState;
+import eutros.multiblocktweaker.crafttweaker.gtwrap.interfaces.IMetaTileEntity;
+import stanhebben.zenscript.annotations.ZenClass;
+
```

Comment:
Add a javadoc here that includes `@zenClass mods.gregtech.predicate.ICandidates` please.